src/Symfony/Component/Scheduler/composer.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "symfony/scheduler",
+  "type": "library",
+  "description": "Symfony Scheduler Component",
+  "keywords": ["schedule", "scheduler", "tasks", "cron"],
+  "homepage": "https://symfony.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Guillaume Loulier",
+      "email": "contact@guillaumeloulier.fr"
+    },
+    {
+      "name": "Symfony Community",
+      "homepage": "https://symfony.com/contributors"
+    }
+  ],
+  "require": {
+    "php": "^7.2.5",
+    "psr/log": "~1.0",
+    "dragonmantank/cron-expression": "^2.3",
+    "symfony/console": "^4.4|^5.0",
+    "symfony/filesystem": "^4.4|^5.0",
+    "symfony/lock": "^4.4|^5.0",
+    "symfony/options-resolver": "^4.4|^5.0",
+    "symfony/process": "^4.4|^5.0",
+    "symfony/serializer": "^4.4|^5.0",
+    "symfony/stopwatch": "^4.4|^5.0"
```

Comment:
for me the stopwatch should be inside require-dev, and event-dispatcher inside require section, no?

Reply:
I'm not sure about this one, the EventDispatcher is not required as if you use Scheduler outside of Sf, you can use it without EventDispatcher, the core "logic" does not require the EventDispatcher to run 🤔

Reply:
The stopwatch has been moved to suggestion, the eventDispatch should stay IMO in the require-dev as the Scheduler can work without it.